src/Microsoft.DocAsCode.Plugins/ISupportIncrementalPostProcessor.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Plugins
+{
+    public interface ISupportIncrementalPostProcessor
+    {
+        /// <summary>
+        /// Get post processor host
+        /// </summary>
+        IPostProcessorHost PostProcessorHost { get; set; }
```

Comment:
post processor won't know its host, and it `should not` set, post processors handler will set host for it. In fact this is to avoid breaking change to update Process method to pass through the host.

---
In reply to: [94203926](https://github.com/dotnet/docfx/pull/1081#discussion_r94203926) [](ancestors = 94203926,94198112,94111780)